networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/network/DelegatingP2PNetwork.java

```diff
@@ -41,6 +43,11 @@ public PeerAddress createPeerAddress(final DiscoveryPeer discoveryPeer) {
     return network.createPeerAddress(discoveryPeer);
   }
 
+  @Override
+  public NodeId parseNodeId(final String nodeId) {
+    return new LibP2PNodeId(PeerId.fromBase58(nodeId));
```

Comment:
This should just delegate to the underlying network
```suggestion
    return network.parseNodeId(nodeId);
```